prepare.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```

It's not entirely POSIX compatible.